backend/src/main.ts

```diff
@@ -2,25 +2,20 @@ import cors from 'cors';
 import express from 'express';
 import http from 'http';
 import * as socketio from 'socket.io';
-import { Game, Player } from './game';
+import { Game } from './game';
+import routes from './routes/routes';
+import sockets from './sockets/sockets';
 
 const app = express();
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(cors());
 
-app.get('/', (req, res) => {
-  res.send({ uptime: process.uptime() });
-});
+const rooms: {
+  [key: string]: Game;
+} = {};
 
-app.get('/rooms', (_req, res) => {
-  res.send(Object.keys(rooms));
-});
-
-app.get('/room/:id', (req, res) => {
-  const room = req.params.id;
-  res.send({ players: rooms[room].players });
-});
+routes(app, rooms);
```

Comment:
suggestion (part 3): and here it would look like this
```suggestion
app.use(roomRouter)
```